chemprop/cli/predict.py

```diff
@@ -307,12 +335,7 @@ def main(args):
 
     model_paths = find_models(args.model_path)
```

Comment:
Maybe we should add `--model-paths` as an alias to `--model-path` to have backwards compatebility and make `--model-paths model1.pt model2.pt` feel more natural. Just a suggestion and feel free to not implement.

Reply:
It totally makes sense to me. I will add it.